bundles/org.openhab.core/src/main/resources/OH-INF/config/addons.xml

```diff
@@ -17,6 +17,12 @@
 				expected. Enabling this option will include these entries in the list of available add-ons.</description>
 			<default>false</default>
 		</parameter>
+		<parameter name="suggestionFinderIp" type="boolean">
```

Comment:
This doesn’t belong in this PR as a change.

Reply:
> You should rebase.

Yeah. This PR got horribly out of synch with main, due to the rapid other PRs prior to v4.1, so I am going to wait until it is finished, and then try to squash/flatten it. (If I can remember to git commands to do that).

Reply:
A rebase on main should eliminate this part. The issue is that you try merging in main.